setup.py

```diff
@@ -64,8 +64,8 @@ def run_tests(self):
     raise Exception('this library does not support python >2 and <3.5')
```

Comment:
I know this is pre-existing, but isn't the exception message here incorrect?

Shouldn't it be:
`this library does not support python >3 and <3.5`

Or perhaps clearer:
`This library does not support Python 3 versions below 3.5.`